pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -134,6 +134,10 @@ type VitessReplicationSpec struct {
 	//
 	// Default: Semi-sync is not enforced.
 	EnforceSemiSync bool `json:"enforceSemiSync,omitempty"`
+
+	// InitializeMaster can be used to enable or disable the operator initializing replication
+	// on a new or restored Vitess Shard object.
+	InitializeMaster bool `json:"initializeMaster,omitempty"`
```

Comment:
I must have missed something - Why is this a different name from `initReplication` which still exists on `VitessKeyspaceSpec`? Did you mean to update all of them to `InitializeMaster`? Also, why is this one no longer a pointer type?

Reply:
`initReplication` still existing on `VitessKeyspaceSpec` was just a mistake. I just cleaned that up.

Reply:
Cool, will read through one more time.